examples/ts/cli.ts

```diff
@@ -236,7 +236,8 @@ const printHumanReadable = (exchange, result) => {
                     return String (x)
                 }
             })
-            log (result.length > 0 ? configuredAsTable (result.map (element => {
+            log (result.length > 0 ? configuredAsTable (result.map (rawElement => {
+                const element = Object.assign ({}, rawElement)
```

Comment:
we just re-define `element` with cloned object and then `info` gets deleted from that cloned object